net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Client.xml

```diff
@@ -18,6 +18,11 @@
                 <pubkey type="TextField">
                     <Required>N</Required>
                 </pubkey>
+                <psk type="TextField">
+                    <Required>N</Required>
+                    <mask>/^[a-z0-9\+\=\/]{44}$/i</mask>
```

Comment:
44 is still weird. do you have an example for me?

Reply:
This example here would match:
https://git.zx2c4.com/WireGuard/about/src/tools/man/wg.8

Reply:
Thanks this checks out... for reference:

    $ openssl rand -base64 32

So we have ... {43}=$ agree ?

Reply:
probably, it is some random value, which is used for the cipher and would fit: https://de.wikipedia.org/wiki/Salsa20

Reply:
@fichtner I agree but this syntax doesn't work .. I'm not a regex specialist, if you like you can try after merge.